site/tsdb-engine/css/demo.css

```
@@ -161,3 +161,562 @@
+}
+.btn-add-filter:hover { border-color: var(--accent); color: var(--accent); }
+
+.hint {
+  font-size: 0.75rem;
+  color: var(--text-secondary);
+  font-weight: 400;
+}
+
+/* ── Results Panel ──────────────────────────────────────────────────── */
+
+.results-panel { }
```

Stylelint (17.7.0)

[error] Empty block (block-no-empty)

(block-no-empty)